frontend/src/main/web/src/components/Row.js

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react'
+import warning from 'warning'
+import { merge } from 'lodash'
+import { View } from './'
+
+const classes = {
+  base: {
+    fld: 'Fld(r)!'
+  },
+  align: {
+    start: {
+      ai: 'Ai(fs)'
+    },
+    end: {
+      ai: 'Ai(fe)'
+    },
+    center: {
+      ai: 'Ai(c)'
+    },
+    baseline: {
+      ai: 'Ai(b)'
+    },
+    stretch: {
+      ai: 'Ai(st)'
+    }
+  }
+}
+
+const Row = ({
```

Comment:
Needs a doc comment. `Row` is a very general name so I want to know a few things from the doc comment:

 - what context I could use this row - do I have to use it inside some other component?
 - is it used for laying out a row of tabular data without actually using a `<table>` element (so just with flexbox or floats or whatever)?
 - is there a good example somewhere of how we have used it?